### ISSUE TITLE
Train Track broken on 4.1 - NoClassDefFoundError: com/hazelcast/jet/IMapJet

### ISSUE THREAD

**Issue by neilstevenson** [hazelcast/hazelcast-jet-demos#117]
```
Exception in thread "main" java.lang.reflect.InvocationTargetException
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.base/java.lang.reflect.Method.invoke(Method.java:564)
    at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
    at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
    at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
    at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
Caused by: java.lang.NoClassDefFoundError: com/hazelcast/jet/IMapJet
    at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
    at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3119)
    at java.base/java.lang.Class.getDeclaredMethod(Class.java:2432)
    at org.apache.beam.sdk.util.InstanceBuilder.buildFromMethod(InstanceBuilder.java:191)
    at org.apache.beam.sdk.util.InstanceBuilder.build(InstanceBuilder.java:155)
    at org.apache.beam.sdk.PipelineRunner.fromOptions(PipelineRunner.java:55)
    at org.apache.beam.sdk.Pipeline.create(Pipeline.java:147)
    at com.hazelcast.jet.demos.traintrack.MyBeamJob.build(MyBeamJob.java:31)
    at com.hazelcast.jet.demos.traintrack.Application.main(Application.java:61)
    ... 8 more
Caused by: java.lang.ClassNotFoundException: com.hazelcast.jet.IMapJet
    at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:466)
    at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:566)
    at org.springframework.boot.loader.LaunchedURLClassLoader.loadClass(LaunchedURLClassLoader.java:92)
    at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:499)
    ... 17 more
```

**Comment by neilstevenson** [comment]
Please assign to me, I know what's wrong. Just dependency changes.

**Comment by neilstevenson** [comment]
No, it's not just dependency changes... looks like the Beam module only works with Jet 3.

**Comment by neilstevenson** [comment]
Should be fixed by Beam 2.21.0

**Comment by neilstevenson** [comment]
Bea, 2.21.0 is out...

**Comment by neilstevenson** [comment]
Fixed by Beam 2.21.0 and https://github.com/neilstevenson/hazelcast-jet-demos/commit/179fe426b72d84acd30a58c68371704e883667ba
